WalletWasabi/Services/Terminate/TerminateService.cs

```diff
@@ -23,9 +23,12 @@ public TerminateService(Func<Task> terminateApplicationAsync, Action terminateAp
 		_terminateApplicationAsync = terminateApplicationAsync;
 		_terminateApplication = terminateApplication;
 		IsSystemEventsSubscribed = false;
-		CancellationToken = TerminationCts.Token; 
+		CancellationToken = TerminationCts.Token;
+		Instance = this;
 	}
 
+	public static TerminateService? Instance { get; private set; }
```

Comment:
A bit strange, but I understand, we need to have a reference somehow if we want to gracefully crash Wasabi at certain points in the software